jcl/src/openj9.gpu/share/classes/com/ibm/gpu/Maths.java

```diff
@@ -32,6 +32,13 @@
  */
 public class Maths {
 
+	/**
+	 * Construct a new instance.
+	 */
+	public Maths() {
```

Comment:
I only see static methods on this class.  Why provide a public constructor?  I'd be tempted to make the class `final` as well.